xtext/org.icyphy.linguafranca/src/org/icyphy/generator/TypeScriptGenerator.xtend

```diff
@@ -264,7 +264,7 @@ class TypeScriptGenerator extends GeneratorBase {
                 "--js_out=import_style=commonjs,binary:" + outPath,
                 "--ts_out=" + srcGenPath)
             protocArgs.addAll(config.protoFiles.fold(newLinkedList, [list, file | list.add(file); list]))
-            val protoc = createCommand("protoc", protocArgs)
+            val protoc = createCommand("protoc", protocArgs, findCommandEnv("protoc"))
```

Comment:
It appears that `findCommandEnv()` is called for every call to `createCommand()`. Also the first argument of `creatCommand()` seems to be always identical to the argument passed to `findCommandEnv()` (unless I missed something). So why not call `findCommandEnv()` directly in `createCommand()`? This would make the usage much simpler.